execution_engine/src/resolvers/v1_resolver.rs

```diff
@@ -160,9 +160,9 @@ impl ModuleImportResolver for RuntimeModuleImportResolver {
                 Signature::new(&[ValueType::I32; 8][..], Some(ValueType::I32)),
                 FunctionIndex::CreateContractUserGroup.into(),
             ),
-            "casper_add_package_version" => FuncInstance::alloc_host(
+            "casper_add_contract_version" => FuncInstance::alloc_host(
                 Signature::new(&[ValueType::I32; 11][..], Some(ValueType::I32)),
```

Comment:
I think this will still not solve the issue -> the import statement embedded in already stored contracts have 10 args, but this one has 11 args. We'd perhaps need two host functions? 

https://github.com/casper-network/casper-node/blob/cb932c54ce34f728e1ce1f1a78b5a0a5affed7c2/execution_engine/src/core/resolvers/v1_resolver.rs#L163-L165

Reply:
Ah makes sense, let me just bring back the old FFI

Reply:
yeah, you need both the old (for retrograde support) and the new (for go forward)